src/v0/branches.ts

```diff
@@ -186,4 +197,36 @@ export class Branches {
       }
     })
   }
+
+  externalId(query: ExternalCustomIdQuery): Promise<BranchResponse> {
```

Comment:
Typically it's best naming practice to have api calls to be an action. Like getExternalId.